polymer-siren-entity-component-wrapper.js

```diff
@@ -0,0 +1,50 @@
+import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+const href = 'href';
+const hrefId = '#' + href;
+const token = 'token';
+const tokenId = '#' + token;
+
+class PolymerSirenEntityComponentWrapper extends PolymerElement {
+	static get is() {
+		return 'polymer-siren-entity-component-wrapper';
+	}
+
+	static get template() {
+		return html`
+			<label for=href>Href:</label>
+			<input id=href type="text" on-change="onChange"/>
+			<label for=token>Token:</label>
+			<input id=token type="text" on-change="onChange"/>
+			<slot></slot>
+		`;
+	}
+
+	static get properties() {
+		return {};
+	}
+
+	get child() {
+		return this.shadowRoot.querySelector('slot')
+			.assignedNodes()

```

Comment:
I think you can use `assignedElements()`. This should allow you to remove the whitespace nodes.